### PR TITLE
Fixing hero Section 

### LINKED ISSUE #124: quick fix for her section in some media query
**quick fix for her section in some media query** 

quickfix

### PR DESCRIPTION
**Pull request by YassinAbdulrahman**
- hero section responsive error done

**_Waiting for feedback Before Merging_** 

newfix

### CODE REVIEW

**Comment by YassinAbdulrahman** [comment]
Waiting for feedback Before Merging ❌❌❌❌.

**Review by AhmadHRai — APPROVED** [review]

**Review by MariaAbdullah97 — APPROVED** [review]

**Pull request merged**
